.github/workflows/beta-deployment.yml

```diff
@@ -14,32 +14,45 @@ on:
       - main
   workflow_dispatch:
 
+
 jobs:
-  buildandtest:
-    name: Build and Test
-    uses: ./.github/workflows/build-and-test.yml
-    permissions:
-      contents: read
-    secrets: inherit
-  iosapptestflightdeployment:
-    name: iOS App TestFlight Deployment
-    needs: buildandtest
-    uses: StanfordBDHG/.github/.github/workflows/xcodebuild-or-fastlane.yml@v2
-    permissions:
-      contents: read
-    with:
-      runsonlabels: '["macOS", "self-hosted"]'
-      googleserviceinfoplistpath: 'PAWS/Supporting Files/GoogleService-Info.plist'
-      setupsigning: true
-      fastlanelane: beta
-    secrets: inherit
+  # buildandtest:
+  #   name: Build and Test
+  #   uses: ./.github/workflows/build-and-test.yml
+  #   permissions:
+  #     contents: read
+  #     actions: read
+  #     security-events: write
+  #   secrets: inherit
+  # iosapptestflightdeployment:
+  #   name: iOS App TestFlight Deployment
+  #   needs: buildandtest
+  #   uses: StanfordBDHG/.github/.github/workflows/xcodebuild-or-fastlane.yml@v2
+  #   permissions:
+  #     contents: read
+  #   with:
+  #     runsonlabels: '["macOS", "self-hosted"]'
+  #     environment: staging
+  #     googleserviceinfoplistpath: 'PAWS/Supporting Files/GoogleService-Info.plist'
+  #     setupsigning: true
+  #     fastlanelane: beta
+  #   secrets: inherit
```

Comment:
TODO: Uncomment these elements once @garricko has tested the Firebase deployment. Uncommented to ensure that the testing doesn't first run the complete iOS deployment.